src/components/sortable-table/sortable-table.component.coffee

```diff
@@ -33,6 +34,9 @@ angular.module('mnoUiElements')
       rowCollectionChanges = _.get(changes, 'rowCollection.currentValue')
       ctrl.displayedCollection = angular.copy(rowCollectionChanges) if rowCollectionChanges
 
+    ctrl.$onInit = () ->
+      ctrl.pipeCb = if ctrl.pipe? then (tableState) -> ctrl.pipe({tableState}) else undefined
```

Comment:
This should work
`ctrl.pipeCb = (tableState) -> ctrl.pipe({tableState}) if ctrl.pipe?`

Reply:
👍

Reply:
After a test I found out it breaks when the `pipe` binding isn't provided - the columns won't sort automatically